src/content-helper/dashboard-page/pages/settings/page-component.tsx

```diff
@@ -1,13 +1,19 @@
+import { PageBody, PageContainer, PageHeader } from '../../components';
+
 /**
  * Settings page component.
  *
  * @since 3.18.0
  */
 export const SettingsPage = () => {
 	return (
-		<>
-			<h1>Parse.ly Settings</h1>
-			<p>This is a page for settings.</p>
-		</>
+		<PageContainer name={ 'settings' }>
```

Comment:
I think we don't need the braces here?

```suggestion
		<PageContainer name="settings">
```